### PR TITLE
fix bugs on landing page code, make the url an easter egg

### PR DESCRIPTION
**Pull request by lacker**
This is just improving a bit of lameness on the homepage - Devin pointed out the <>'s don't work within a Text tag, so I removed them, and someone else pointed out that nonexistent fake urls are suboptimal, so I improved that too.

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @sunnylqm to be a potential reviewer.

File: website/src/react-native/index.js [hunk 1]
```diff
@@ -50,8 +50,8 @@ class WhyReactNativeIsSoGreat extends Component {
           If you like React on the web, you'll like React Native.
         </Text>
         <Text>
-          You just use native components like '<View>' and '<Text>',
-          instead of web components like '<div>' and '<a>'.
+          You just use native components like 'View' and 'Text',
```
Comment (JoelMarcey) [inline website/src/react-native/index.js]: No way to escape them?

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-3** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/158284351251685/int_phab) to review.

**Pull request closed without merging**